### PR TITLE
Noise refactor4

### PR DESCRIPTION
**Pull request by efidoman**

### CODE REVIEW

File: bin/g5-calc-noise.sh [hunk 1]
```diff
@@ -87,26 +88,26 @@ do
...
   #echo "sod=$sod, noise=$noise, absDelta=$absDelta"
 
   lastDelta=$delta
 done
 
 # to ensure mostly straight lines with small bounces don't give heavy noise
-  if [ $(bc -l <<< "$noise > $LIGHT_MAX_NOISE") -eq 1 ]; then
-    if [ $(bc -l <<< "($sod / $n) < 3") -eq 1 ]; then
+  if [ $(bc -l <<< "$noise > $CLEAN_MAX_NOISE") -eq 1 ]; then
+    if [ $(bc -l <<< "($sod / $n) < 6") -eq 1 ]; then
```
Comment (tynbendad) [inline bin/g5-calc-noise.sh]: do you want to set a constant for this magic value? i'm not sure what it represents...

File: bin/g5-calc-noise.sh [hunk 2]
```diff
@@ -87,26 +88,26 @@ do
...
      noise=$CLEAN_MAX_NOISE # very small up/downs shouldn't cause noise 
-    elif [ $(bc -l <<< "($sod / $n) < 6") -eq 1 ]; then
+    elif [ $(bc -l <<< "($sod / $n) < 9") -eq 1 ]; then
```
Comment (tynbendad) [inline bin/g5-calc-noise.sh]: same here

**Comment by tynbendad** [comment]
i changed RISE_WITHOUT_ADDED_NOISE to 12 and the "Light" noises went back to Clean.

**Comment by efidoman** [comment]
Good testing and definitely a valid rise scenario for high carb meals. We'll go with RISE_WITHOUT_ADDED_NOISE = 12

File: bin/g5-calc-noise.sh (hunk 1 above)
Comment (efidoman) [reply bin/g5-calc-noise.sh]: Yes, I'll call them CLEAN_MAX_AVERAGE_DELTA, and LIGHT_MAX_AVERAGE_DELTA

File: bin/g5-calc-noise.sh (hunk 2 above)
Comment (efidoman) [reply bin/g5-calc-noise.sh]: Yes, will label them. Good point.

**Comment by tynbendad** [comment]
Great, I will keep watching it as always and see if it needs to be higher.

**Pull request merged**